explorer/src/components/account/BlockHistoryCard.tsx

```diff
@@ -0,0 +1,143 @@
+import React from "react";
+import { TableCardBody } from "components/common/TableCardBody";
+import { useBlock, useFetchBlock, FetchStatus } from "providers/block";
+import { Signature } from "components/common/Signature";
+import { ErrorCard } from "components/common/ErrorCard";
+import { LoadingCard } from "components/common/LoadingCard";
+import { Block } from "components/common/Block";
+
+export function BlockHistoryCard({ block }: { block: string }) {
+  const blockData = useBlock(block);
+  const fetchBlock = useFetchBlock();
+  const refresh = () =>
+    fetchBlock(block, isNaN(Number(block)) ? "blockhash" : "slot");
```

Comment:
We can only support slot queries with the current RPC API, can you please remove the blockhash logic?